python/ray/air/callbacks/wandb.py

```diff
@@ -183,11 +183,16 @@ def __init__(
         queue: Queue,
         exclude: List[str],
         to_config: List[str],
+        flush_interval: int,
         *args,
         **kwargs,
     ):
         super(_WandbLoggingProcess, self).__init__()
 
+        import wandb
+
+        self._wandb = wandb
```

Comment:
this will cause serialisation issues, we need to add setstate and getstate that can deal with it (remove from state dict and import again and set on load)

Reply:
Can you elaborate more? When will `_WandbLoggingProcess` be serialized?

Reply:
I don't think that will happen, but it is still bad practice IMO to have unserializable arguments when we can easily avoid it. Very error prone

Reply:
updated to prevent pickleable